packages/protocol/test/common/accounts.ts

```diff
@@ -350,16 +362,16 @@ contract('Accounts', (accounts: string[]) => {
   describe('#batchGetMetadataURL', () => {
     it('returns multiple metadata URLs', async () => {
       const randomStrings = accounts.map((_) => web3.utils.randomHex(20).slice(2))
-      await Promise.all(
-        accounts.map(async (mappedAccount, i) => {
-          await accountsInstance.createAccount({ from: mappedAccount })
-          await accountsInstance.setMetadataURL(randomStrings[i], { from: mappedAccount })
-        })
-      )
+
+      for (let i = 0; i < accounts.length; i++) {
+        await accountsInstance.createAccount({ from: accounts[i] })
+        await accountsInstance.setMetadataURL(randomStrings[i], { from: accounts[i] })
+      }
```

Comment:
Changed to a regular for loop, as the `promise.all()` would timeout. 

My guess is that this is caused by transactions not being submitted in order.